docs/Users_Guide/tc-diag.rst

```diff
@@ -204,8 +213,16 @@ The **vortex_removal** flag entry is a boolean specifying whether or not vortex
 
 .. note:: As of MET version 11.1.0, vortex removal logic is not yet supported.
 
-Configuring data output options
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Configuring data input and output options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+  one_time_per_file_flag = TRUE;
+
+The **one_time_per_file_flag** entry controls the logic for reading data from input files. Users should set this to true or false to describe how data is stored in the gridded input files specified with the **-data** command line option. Set this to true if each input file contains all of the data for a single initialization time and for a single valid time. If the input files contain data for multiple initialization or valid times, or if data for one valid time is spread across multiple files, set this to false.
```

Comment:
The -data snippet is spread across two lines in the rendered doc. Is there any way to protect this term from spreading across the line break? I couldn't find out how after a few minutes of searching . . .

Reply:
It would seem that RTD treats the leading `-` and the following `data` as part of separate words when wrapping text. I'm guessing it treats the same as hyphenated words like "check-in" where the `-` character is a good place to wrap. You'll notice that as you adjust the browser window width, the wrapping changes dynamically.

I reworded the sentence to address this by moving the the `-data` away from the end of the line when the browser window is at max width. I could switch the formatting from bold **-data** to this in-line code block `-data` instead. That'd probably fix it but wouldn't be consistent with the formatting in the rest of the User's Guide.

But we could choose to modify the formatting for all `-something` command line options to always use that in-line code block formatting... just not in this PR.

Reply:
Actually, playing around with the formatting of my comment above by modifying the browser window width, formatting as `-data` does NOT fix it. It still wraps at the `-` as it would for hyphenated words. Probably not something worth worrying about any more than this.